src/App/ImportUshahidiV2/Jobs/CreateDefaultSurvey.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+namespace Ushahidi\App\ImportUshahidiV2\Jobs;
+
+use Illuminate\Support\Facades\Log;
+use Ushahidi\Core\Entity;
+use Ushahidi\App\Jobs\Job;
+use Ushahidi\App\ImportUshahidiV2;
+
+class CreateDefaultSurvey extends Job
+{
+    protected $importId;
+
+    /**
+     * Create a new job instance.
+     *
+     * @return void
+     */
+    public function __construct($importId)
+    {
+        $this->importId = $importId;
+    }
+
+    // Add default attributes
+    protected $defaultAttributes = [
+        [
+            'key' => 'title',
+            'input' => 'text',
+            'label' => 'Report Title',
+            'priority' => 1,
+            'required' => true,
+            'type' => 'title',
+            'options' => [],
+            'config' => [],
+            'cardinality' => 1,
+        ],
+        [
+            'key' => 'description',
+            'input' => 'text',
+            'label' => 'Description',
+            'priority' => 2,
+            'required' => true,
+            'type' => 'description',
+            'options' => [],
+            'config' => [],
+            'cardinality' => 1,
+        ],
+        [
+            'key' => 'date',
+            // 'original_id' => 'date',
```

Comment:
remove?

Reply:
Yes. It's mostly just for my own reference until I get posts importing and can figure out if I need it somehow